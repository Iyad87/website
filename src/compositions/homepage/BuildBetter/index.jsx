import React from 'react';

const BuildBetter = () => null;

export default BuildBetter;
