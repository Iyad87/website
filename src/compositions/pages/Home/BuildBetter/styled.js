import styled from 'styled-components';
import { highlight, text } from 'styles/colours';
import { spacing } from 'styles/spacing';
import { DefaultTitle } from 'components/PageHero/styled';

export const Title = DefaultTitle.extend`
  color: ${text};
  font-weight: 500;
  font-size: 4.5rem;
  margin-bottom: ${spacing(4)};
  margin-top: ${spacing(4)};

  &:after {
    content: '.';
    color: ${highlight};
  }
`;

export const Paragraph = styled.p`
  margin: ${spacing(4)} 0;
`;
