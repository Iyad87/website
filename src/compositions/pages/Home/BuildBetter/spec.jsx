import React from 'react';
import { shallow } from 'enzyme';
import BuildBetter from './index';

test('compositions/homepage/BuildBetter snapshot', () => {
  const wrapper = shallow(<BuildBetter />);

  expect(wrapper).toMatchSnapshot();
});
