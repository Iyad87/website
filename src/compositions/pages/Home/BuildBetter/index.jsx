import React from 'react';

import Container from '../../../../components/Container';
import { Title, Body } from './styled';

const BuildBetter = () => (
  <Container>
    <Title>Build better</Title>
    <Body>
      Great companies go beyond their customers expectations, over and over
      again. Building this capability in your company is our mission, and our
      promise.
    </Body>
    <Body>
      We enable your organisation to deliver world-class technology and customer
      experiences.
    </Body>
  </Container>
);

export default BuildBetter;
