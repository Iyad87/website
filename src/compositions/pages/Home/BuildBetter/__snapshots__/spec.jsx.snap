// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/homepage/BuildBetter snapshot 1`] = `
<Container>
  <styled__Title>
    Build better
  </styled__Title>
  <styled__Body>
    Great companies go beyond their customers expectations, over and over again. Building this capability in your company is our mission, and our promise.
  </styled__Body>
  <styled__Body>
    We enable your organisation to deliver world-class technology and customer experiences.
  </styled__Body>
  <Button
    component="a"
    href="#case-studies"
  >
    Case Studies â†“
  </Button>
</Container>
`;
